src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceGeneral3.java

```diff
@@ -0,0 +1,88 @@
+/*
+VariableDeclarationUsageDistance
+allowedDistance = 1
+ignoreVariablePattern = (default)
+validateBetweenScopes = true
+ignoreFinal = false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.variabledeclarationusagedistance;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputVariableDeclarationUsageDistanceGeneral3 {
+
+    void nothing() {
+    }
+
+    class Parent {
+        void mm() {
+        }
+    }
+
+    public void method2() {
+        // false positive, the distance is 1
```

Comment:
If we know that it bad behavior, We usually define 'until #xxxxxx' in comment to make it clear that we have issue on this

Reply:
Does it mean that every false positive and false negative should have issues created?

Reply:
Ideally Yes, but we can create one issue, with hope to fix all in one.  During fix PR we can decide to split more or ...

Reply:
@jxr98, ping

Reply:
Yes, I am creating a new issue. And I want to merge PR#12822 before this because these two prs modified a same file.

Reply:
I am ready to merge this PR.
I recommend to merge this first and let bigger PR continue review, it might take a bit more time

Reply:
Put comment "until #xxxx" in inputs to make it clear this is temporary behavior .

And let's merge this PR

Reply:
okay. Will take a few minutes.